web/src/components/merchant/sign-up/index.tsx

```diff
@@ -0,0 +1,40 @@
+/**
+ * Copyright 2020 Google LLC
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     https://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+import React from 'react';
+
+import SignUpCard from 'components/merchant/sign-up/SignUpCard';
+import Container from 'muicss/lib/react/container';
+import styled from 'styled-components';
+
+const PageContainer = styled(Container)`
+  max-height: 800px;
+  height: 100vh;
```

Comment:
```suggestion
  min-height: 100vh;
  height: 100%;
```
Hmm why not this? This way the sign up card will always be centralised and when content is longer than 100vh you can scroll too.

Reply:
I think when it's centralised in the large monitor it's too low. That's why I set the max-height.

Reply:
Like I literally have to move my head to look at the form from the title of the tab :o

Reply:
Okay actually I think I will make the font-size and all the sizes responsive for both large monitor and mobile in a separate PR.